Rakefile

```diff
@@ -266,6 +267,7 @@ task :ci do
       sh 'bundle exec appraisal contrib-old rake spec:faraday'
       sh 'bundle exec appraisal contrib-old rake spec:http'
       sh 'bundle exec appraisal contrib-old rake spec:mongodb'
+      sh 'bundle exec appraisal contrib-old rake spec:mysql2'
```

Comment:
Once the prepend compatibility is implemented could you add the same line to L230. Thanks!